{{cookiecutter.extension_name}}/RELEASE.md

```diff
@@ -0,0 +1,30 @@
+# Making a {{ cookiecutter.extension_name }} release
+
+This document guides a contributor through creating a release of {{ cookiecutter.extension_name }}.
+
+## Check installed tools
+
+### Clean the repository
+
+To ensure a clean build, you can remove all non-tracked files with:
```

Comment:
... build, remove all non-tracked files using the following command from the repository's root directory: